lib/Controller/SiteController.php

```diff
@@ -100,6 +100,7 @@ protected function createResponse(int $id, array $site, string $path = ''): Temp
 		$policy = new ContentSecurityPolicy();
 		$policy->addAllowedWorkerSrcDomain('*');
 		$policy->addAllowedFrameDomain('*');
+		$policy->addAllowedFrameDomain('blob:');
```

Comment:
How is this not covered by the `*` already?

Reply:
Because `*` only includes `http` AFAICT, so for non `http` stuff you need to explicitly specify the protocol.

Not sure if there are others that make sense, but for immich `blob` is needed.